fiasco/util/exceptions.py

```diff
@@ -1,7 +1,9 @@
 """
-Custom exceptions
+Custom exceptions for fiasco.
 """
 
+__all__ = ['MissingIonError', 'MissingDatabaseError', 'MissingASCIIFileError',
+              'MissingDatasetException', 'UnsupportedVersionError']
```

Comment:
```suggestion
__all__ = [
    'MissingIonError', 
    'MissingDatabaseError', 
    'MissingASCIIFileError',
    'MissingDatasetException', 
    'UnsupportedVersionError'
]
```

The indentation here was weird. Seems sensible to put these all on separate lines?

Reply:
Yeah.